### PR TITLE
feat(image): created web component image component

### PR DESCRIPTION
**Pull request by andysherman2121**
### Related Ticket(s)

https://github.com/carbon-design-system/ibm-dotcom-library/issues/3540

### Description

Create image web component

### Changelog

**New**

adds component, test and story

### CODE REVIEW

File: packages/web-components/src/components/image/__stories__/image.stories.ts [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+// import Add16 from 'carbon-custom-elements/es/icons/add/16';
```
Comment (asudoh) [inline packages/web-components/src/components/image/__stories__/image.stories.ts:11]: Did you still want to add icon example, or do you want to remove this line?

File: packages/web-components/src/components/image/__tests__/image.test.ts [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    it('should render with various attributes', async function() {
+      render(
+        template({
+          defaultSrc: 'https://dummyimage.com/672x336/ee5396/161616&text=2x1',
+          alt: 'Image alt text',
+        }),
+        document.body
+      );
+      await Promise.resolve(); // Update cycle for `<dds-feature-card>`
```
Comment (asudoh) [inline packages/web-components/src/components/image/__tests__/image.test.ts:39]:
```suggestion
      await Promise.resolve(); // Update cycle for `<dds-image>`
```

**Review by asudoh — APPROVED** [review]
LGTM 👍🎉 - Thanks @andysherman2121!

File: packages/web-components/src/components/image/__tests__/image.test.ts (hunk 2 above)
Comment (andysherman2121) [reply packages/web-components/src/components/image/__tests__/image.test.ts]: fixed!

File: packages/web-components/src/components/image/__stories__/image.stories.ts (hunk 1 above)
Comment (andysherman2121) [reply packages/web-components/src/components/image/__stories__/image.stories.ts]: Good catch, i'm gonna remove this

File: packages/web-components/src/components/image/__stories__/README.stories.mdx [hunk 3]
```diff
@@ -0,0 +1,42 @@
+import { Preview, Props, Description, Story } from '@storybook/addon-docs/blocks';
+import contributing from '../../../../../../docs/contributing-license.md';
+import Image from '../Image';
+
+# Image
+
```
Comment (jeffchew) [inline packages/web-components/src/components/image/__stories__/README.stories.mdx:6]: Hey @andysherman2121 , great PR! Want to suggest adding a short description of what this is, which is essentially a responsive image. May also want to describe what image items are and how to configure each with its own breakpoint. The example you have below doesn't have that at all so might not be as much value there, perhaps have an example with the media queries configured.

Comment (jeffchew) [reply packages/web-components/src/components/image/__stories__/README.stories.mdx:6]: Also perhaps an example with the description text as well.

File: packages/web-components/src/components/image/image.ts [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    return html`
+      <slot @slotchange="${handleSlotChange}"></slot>
+      <picture>
+        ${images.map(
+          image => html`<source media="${image.getAttribute('media')}" srcset="${image.getAttribute('srcset')}"></source>`
+        )}
+        <img class="${prefix}--image__img" src="${defaultSrc}" alt="${alt}" aria-describedby="long-description" />
+      </picture>
+      <div id="long-description" class="${prefix}--image__longdescription">
+        <slot name="long-description"></slot>
```
Comment (jeffchew) [inline packages/web-components/src/components/image/image.ts:63]: Can we maybe have a story in storybook that includes the image description too?

File: packages/web-components/src/components/image/__stories__/README.stories.mdx (hunk 3 above)
Comment (andysherman2121) [reply packages/web-components/src/components/image/__stories__/README.stories.mdx:6]: updated

**Pull request merged**